### ISSUE TITLE
trained 60 epoch,the validation loss of paf and heatmap is still decrease

### ISSUE THREAD

**Issue by hudengjunai** [michalfaber/tensorflow_Realtime_Multi-Person_Pose_Estimation#3]
hello,I pull down your code almost no change.I trained the mobilenet based model on 2080Ti with 60 epoch.and I found the val_loss is still decrease
![7DA694CA-9D1A-41EB-9E27-8C189367E857](https://user-images.githubusercontent.com/33210026/70961256-40b89300-20bc-11ea-8c9d-f55756c05c6a.png)
the x-axis is the epoch from 1 to 60,the y-axis is loss.

**Comment by michalfaber** [comment]
That's good. The lower val loss, the better.

**Comment by af-yi** [comment]
Excuse me,Are you using the same learning strategy as train_custom_loop_mobilenet.py?
